packages/modules/src/modules/banners/choiceCardsButtonsBanner/choiceCardsButtonsBannerStyles.ts

```diff
@@ -60,6 +60,16 @@ export const columnMarginOverrides = css`
     }
 `;
 
+export const eol_choiceCardverticlaAlignment = css`
```

Comment:
Minor point, but what does the `eol_` prefix on this variable name signify & do we need it? Also there's a couple typos/casing discrepancies in the variable name "choiceCard**v**erticl**a**Alignment".

Reply:
hey ... the eol was just a reminder that this ought to be short lived (end of life). I can remove it though. And I should definitely fix my horrible spelling!